### PR TITLE
Fallbacks for getting token prices

### LINKED ISSUE #7675: Failed to fetch price from Coingecko
image

### PR DESCRIPTION
**Pull request by shashkovdanil**
## PR Type

Improvement

## Context

Closes #7675

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP&usdamount=0)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
     "@kodadot1/sub-api": "0.2.0-rc.0",
     "@nuxtjs/apollo": "5.0.0-alpha.6",
     "@nuxtjs/i18n": "8.0.0-rc.3",
+    "@nuxtjs/proxy": "^2.1.0",
```
Comment (roiLeo) [inline package.json:72]: we doesn't need this one, use `routeRules` or `nitro` config

```suggestion
    "@nuxtjs/proxy": "^2.1.0",
```

**Review by roiLeo — COMMENTED** [review]
related with kodadot/workers#195?

**Comment by preschian** [comment]
Hi @shashkovdanil, thanks for the contribution. but, so sorry 🙏, I just made an adjustment here:
- https://github.com/kodadot/workers/pull/195
- https://github.com/kodadot/nft-gallery/pull/8036

**Comment by yangwao** [comment]
> but, so sorry 🙏, I just made an adjustment here:

Made comment here
- https://github.com/kodadot/nft-gallery/pull/8036#issuecomment-1807790967

Would happily reuse that exchange tickers on workers, which @shashkovdanil made with payout to him, as current coniprice worker use only sub.id which I see as SPOF currently. 
Fallbacks supposed to be on workers, not on client side as it will fire too much requests from client

**Comment by preschian** [comment]
> Would happily reuse that exchange tickers on workers, which @shashkovdanil made with payout to him, as current coniprice worker use only sub.id which I see as SPOF currently.
> Fallbacks supposed to be on workers, not on client side as it will fire too much requests from client

aahh, yes, good point here 👍

> Fallbacks supposed to be on workers

👍👍

**Comment by preschian** [comment]
@yangwao @shashkovdanil let's close this PR? and create a new PR from this PR #8036 to add Binance API to this line https://github.com/kodadot/nft-gallery/blob/feat/kodaprice/utils/coinprice.ts#L40-L61

I already added kraken API to the workers side https://github.com/kodadot/workers/blob/5b66d04c8a0a0d5aabc687825e2745e005a22a50/coinprice/src/utils.ts#L44

**Comment by shashkovdanil** [comment]
Is there any action required of me?

**Comment by yangwao** [comment]
> Is there any action required of me?

please update your payout address with DOT address in this PR description 

Go to https://canary.kodadot.xyz/dot/transfer

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 5.49 USD/DOT ~ 9.107 $DOT
🧗 [16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP](https://polkadot.subscan.io/account/16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP)
🔗 [0x38711e7f363f7d2bd4ac0e2f51b801c3842712f1ec3e05670ac8b83cc8cd9132](https://polkadot.subscan.io/extrinsic/0x38711e7f363f7d2bd4ac0e2f51b801c3842712f1ec3e05670ac8b83cc8cd9132)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by preschian** [comment]
> merged commit [9b3ef3f](https://github.com/kodadot/nft-gallery/commit/9b3ef3f531750eacaa365e295a4b927bd705023c) into kodadot:main

why merge 😅🤣

**Comment by roiLeo** [comment]
oh no..

**Pull request merged**